src/main/java/uk/gov/hmcts/reform/bulkscanning/controllers/SasTokenController.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.bulkscanning.controllers;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.bulkscanning.model.out.SasTokenResponse;
+import uk.gov.hmcts.reform.bulkscanning.services.SasTokenGeneratorService;
+
+@RestController
+public class SasTokenController {
+
+    private final SasTokenGeneratorService tokenGeneratorService;
+
+    public SasTokenController(SasTokenGeneratorService tokenGeneratorService) {
+        this.tokenGeneratorService = tokenGeneratorService;
+    }
+
+    @GetMapping(path = "/{serviceName}", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I would add a prefix to avoid even accidental name clashes. Anything would do, `sas`, `token`, or `services`, or `keys`, or... you get the point :grin: